sci-mathematics/nusmv/nusmv-2.6.0.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils flag-o-matic toolchain-funcs
+
+MY_P="NuSMV-${PV}"
+NUSMV_A="${MY_P}.tar.gz"
+MINISAT_A="37dc6c67e2af26379d88ce349eb9c4c6160e8543.zip"
+ZCHAFF_A="zchaff.64bit.2007.3.12.zip"
+
+DESCRIPTION="NuSMV: new symbolic model checker"
+HOMEPAGE="http://nusmv.fbk.eu/"
+SRC_URI="http://nusmv.fbk.eu/distrib/${NUSMV_A}
+		minisat? ( https://github.com/niklasso/minisat/archive/${MINISAT_A} )
+		zchaff? ( http://www.princeton.edu/~chaff/zchaff/${ZCHAFF_A} )"
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="minisat doc zchaff"
+REQUIRED_USE="|| ( minisat zchaff )"
+
+RDEPEND="dev-libs/expat"
+DEPEND="${RDEPEND}
+		doc? (
+			virtual/latex-base
+			dev-texlive/texlive-latexextra
+			app-doc/doxygen
+		)
+		>=dev-util/cmake-2.8
+		dev-lang/python:2.7
+		dev-libs/libxml2"
+
+S="${WORKDIR}/${MY_P}/NuSMV"
+
+DOCS=( AUTHORS NEWS README{,_MiniSat,_PLATFORMS,_zChaff}.txt )
+
+src_unpack() {
+	unpack ${NUSMV_A}
+	if use minisat; then
+		cp "${DISTDIR}/${MINISAT_A}" "${WORKDIR}/${MY_P}/MiniSat" || die
+	fi
+	if use zchaff; then
+		cp "${DISTDIR}/${ZCHAFF_A}" "${WORKDIR}/${MY_P}/zChaff" || die
+	fi
+}
+
+src_prepare() {
+	sed -i -e 's/-mcpu=[^\s]*//' "${WORKDIR}/${MY_P}/cudd-2.4.1.1"/Makefile || die
+	# Change the "prog-man/html" rule to optional
+	sed -i -e '/\s*install( .*html /s:${PROJECT_BINARY_DIR}:share/nusmv/doc OPTIONAL:' "${S}"/doc/prog-man/CMakeLists.txt || die
+	# Change the doc destination
+	sed -i -e "s:share/nusmv/doc:share/doc/${PF}:" "${S}"/doc/{tutorial,user-man,prog-man}/CMakeLists.txt || die
+
+	# Correction for proper parallel compilation
+#	sed -i -e 's/COMMAND ${MAKE}/COMMAND $(MAKE)/' "${WORKDIR}/${MY_P}/"{zchaff,MiniSat}/CMakeLists.txt || die
+	default
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DPYTHON_EXECUTABLE:FILEPATH=/usr/bin/python2.7
+		-DCMAKE_INSTALL_PREFIX=/usr
+		-DENABLE_MINISAT=$(usex minisat)
+		-DENABLE_ZCHAFF=$(usex zchaff)
+	)
+
+	cmake-utils_src_configure
+}
+
+src_compile() {
+	local targets=( all )
+	use doc && targets+=( prog-man user-man html tutorial docs )
+	cmake-utils_src_compile "${targets[@]}"
+}
+
+src_install() {
+	cmake-utils_src_install
+	# Remove docs where they do not belong to
+	rm -f "${D}"/usr/share/nusmv/{LGPL-2.1,README*,NEWS} || die
```

Comment:
use `"${D%/}"` instead of `"${D}"` to avoid double slash. Consider using `ED` instead, as that will make prefixing it easier.